docs/src/example.md

```diff
@@ -72,7 +29,14 @@ end
 Completely equivalent implementation is
 
 ```julia
-AtomsCalculators.@generate_interface function AtomsCalculators.calculate(::AtomsCalculators.Energy, system, calculator::MyType; kwargs...)
+AtomsCalculators.@generate_interface function AtomsCalculators.calculate(
+    ::AtomsCalculators.Energy, 
+    system, 
+    calculator::MyType,
+    ps=nothing,
+    st=nothing; 
```

Comment:
I'm not too happy about abbreviating these. I think it does not hurt to keep the long form for clarity.

Reply:
In general, I might agree but `ps, st` seem to be standard throughout the Lux eco-system. My mild preference is to follow this standard.